packages/tests/package.json

```diff
@@ -6,9 +6,14 @@
     "codegen:bigBoi": "tsx ../../scripts/generateBigBoi.ts",
     "codegen:bigLegacyRouter": "tsx ../../scripts/generateBigLegacyRouter.ts",
     "lint": "eslint --ext .ts,.tsx --ignore-path ../../.gitignore --report-unused-disable-directives .",
-    "test": "jest",
-    "test-watch": "jest --watch",
-    "test-ci": "jest --ci --maxWorkers=2 --coverage"
+    "test": "concurrently -c \"green,blue\" npm:test-run:*",
+    "test-run:vitest": "vitest --run",
+    "test-run:tsc": "tsc --noEmit --pretty",
```

Comment:
Why not setup typechecking via Vitest: https://vitest.dev/config/#typecheck

Reply:
I tried but it doesn't work and doesn't provide much value since you still have to run it as a separate command - `vitest typecheck`